include/exadg/incompressible_navier_stokes/spatial_discretization/operators/momentum_operator.cpp

```diff
@@ -59,11 +60,38 @@ MomentumOperator<dim, Number>::initialize(
 
   if(operator_data.viscous_problem)
   {
+    this->viscous_kernel_own_storage = true;
```

Comment:
if we call this `initialize()` with this signature, we are in multigrid initialize.
we need to setup the `viscous_kernel` in this case and therefore also  setup the variable viscosity models following the current design of `master`